src/pinning.jl

```diff
@@ -125,6 +127,7 @@ function getcpuids_pinning(::RandomBind, places; kwargs...)
     reduce(vcat, Random.shuffle(getplaces_cpuids(places)))
 end
 getcpuids_pinning(::CurrentBind, args...; kwargs...) = getcpuids()
+getcpuids_pinning(::FirstNBind, args...; kwargs...) = cpuids_all()[1:nthreads()]
```

Comment:
Might need a bounds check. For example, `nthreads()` could be larger than the number of available cpu ids.